dict/portuguese.yml

```diff
@@ -2,290 +2,290 @@
 
   MIDDLE:
     'BZ': [Belize, Belizenh*, Belizense*, Belmopan, Belmopã]
-    'CR': [Costa Rica, República da Costa Rica, Costa-Riquenh*, Costa-Riquense*, Costarriquense*, Costa-Ricense*, Costarricense*, Ticos, San José, São José]
-    'GT': [Guatemala, República da Guatemala, Guatemaltec*, Guatemalense*, Guatemalês*, Guatemales*, Cidade da Guatemala]
-    'HN': [Honduras, República das Honduras, República de Honduras, Hondurenh*, Tegucigalpa]
+    'CR': [Costa Rica, Costa-Riquenh*, Costa-Riquense*, Costarriquense*, Costa-Ricense*, Costarricense*, Ticos, San José, São José]
+    'GT': [Guatemala, Guatemaltec*, Guatemalense*, Guatemalês*, Guatemales*, Cidade da Guatemala]
+    'HN': [Honduras, Hondurenh*, Tegucigalpa]
     'MX': [México, Estados Unidos Mexicanos, Mexican*, Cidade do México]
-    'NI': [Nicarágua, República da Nicarágua, Nicaraguense*, Nica, Manágua]
-    'PA': [Panamá, República do Panamá, Panamenh*, Panamense*, Cidade do Panamá]
-    'SV': [El Salvador, República de El Salvador, Salvadorenh*, San Salvador, São Salvador]
+    'NI': [Nicarágua, Nicaraguense*, Nica, Manágua]
+    'PA': [Panamá, Panamenh*, Panamense*, Cidade do Panamá]
+    'SV': [El Salvador, Salvadorenh*, San Salvador, São Salvador]
 
   SOUTH:
-    'AR': [Argentina, República Argentina, Argentin*, Buenos Aires]
-    'BO': [Bolívia, Estado Plurinacional da Bolívia, Bolivian*, Sucre, La Paz]
-    'BR': [Brasil, República Federativa do Brasil, Brasileir*, Brasília, São Paulo, Rio de Janeiro]
-    'CL': [Chile, República do Chile, Chilen*, Santiago, Santiago do Chile]
-    'CO': [Colômbia, República da Colômbia, Colombian*, Bogotá]
-    'EC': [Equador, República do Equador, Equatorian*, Quito]
+    'AR': [Argentina, Argentin*, Buenos Aires]
+    'BO': [Bolívia, Bolivian*, Sucre, La Paz]
+    'BR': [Brasil, Brasileir*, Brasília, São Paulo, Rio de Janeiro]
+    'CL': [Chile, Chilen*, Santiago, Santiago do Chile]
+    'CO': [Colômbia, Colombian*, Bogotá]
+    'EC': [Equador, Equatorian*, Quito]
     'FK': [Ilhas Malvinas, Ilhas Falkland, Falklands, Falklander*, Malvinense*]
     'GF': [Guiana Francesa, Franco-Guianense*]
-    'GY': [Guiana, Guyana, República Cooperativa da Guiana, Guianês*, Guianense*]
-    'PE': [Peru, República do Peru, Peruan*, Peruvian*, Lima]
-    'PY': [Paraguai, República do Paraguai, Paraguai*, Paraguaian*, Assunção]
-    'SR': [Suriname, República do Suriname, Surinamense*, Surinamês*, Paramaribo]
-    'UY': [Uruguai, República Oriental do Uruguai, Uruguai*, Montevidéu, Montevideu]
-    'VE': [Venezuela, República Bolivariana da Venezuela, Venezuelan*, Caracas, Santiago de León de Caracas]
+    'GY': [Guiana, Guyana, Guianês*, Guianense*]
+    'PE': [Peru, Peruan*, Peruvian*, Lima]
+    'PY': [Paraguai, Paraguai*, Paraguaian*, Assunção]
+    'SR': [Suriname, Surinamense*, Surinamês*, Paramaribo]
+    'UY': [Uruguai, Uruguai*, Montevidéu, Montevideu]
+    'VE': [Venezuela, Venezuelan*, Caracas, Santiago de León de Caracas]
```

Comment:
Do we need "Santiago de León de Caracas"?

Reply:
Same thing as the previous cases.